Autopilot/Relay/AutoRelay_v0_01.c

```diff
@@ -254,9 +279,11 @@ int main (void)
     Init();
 	while(1)
 	{
-
-                //The window on the controller is 43%-50%
-		if ((ch8_position > 400) && (ch8_position < 430)) //Controller Setting is: +47%, -33% //(ch8_position > 450)
+        thisCaptureTime = TMR1;
+        unsigned int timeDiff = (thisCaptureTime - lastCaptureTime)/T1_TICKS_TO_MSEC;
```

Comment:
Because you're not taking into account for timer overflow, this code will produce unexpected results as you're doing a substraction between unsigned ints. Since you're not reseting the timer anywhere, eventually thisCaptureTime  may be smaller than lastCaptureTime, resulting in an unsigned integer overflow. You'll get a huge time diff and the relay will turn to manual when it shouldnt